book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,158 @@
+# Simple Payment and State Verification
+
+Thin clients using Solana require a receipt for transactions that have been
+accepted by the cluster without running a full validator node.  New fullnode
+validators that desire to boot from a known checkpoint in the network require
+some proof that the checkpoint is valid.
+
+Solana receipts provide a proof that a transaction or state have been confirmed
+by the desired set of validators to a desired amount of finality.
```

Comment:
Instead of saying "amount of finality" can we say "number of confirmations"?

Reply:
Magnitude is more accurate for a lot of clients.